CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(PyTorch)
 # Extension meta-information
 set(EXTENSION_HOMEPAGE "https://github.com/fepegar/SlicerPyTorch")
 set(EXTENSION_CATEGORY "Utilities")
-set(EXTENSION_CONTRIBUTORS "Fernando Perez-Garcia (University College London and King's College London) and Andras Lasso (PerkLab Queen's University)")
+set(EXTENSION_CONTRIBUTORS "Fernando Pérez-García (University College London and King's College London), Andras Lasso (PerkLab Queen's University)")
```

Comment:
The extension catalog web frontend shows mojibake for this accented character. Should we still change this? How code page is specified for CMake files? Is it always utf-8? Or the system code page? Or something else?

Reply:
Thanks for the review, I will follow-up on this later tomorrow.